components/gitpod-protocol/src/blocked-repositories-protocol.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+// TODO: Maybe this should be in protocol.ts?
```

Comment:
> // TODO: Maybe this should be in protocol.ts?

Maybe it's fine to have one top-level file per (DB) entity. If anything I'd try to group it with other abuse-related shapes (if there is opportunity).